lib/v2/lisa/radar.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+    'sonymusic.co.jp': {
+        _name: 'Sony Music',
+        www: [
+            {
+                title: 'LiSA News',
+                docs: 'https://docs.rsshub.app/other.html#lisa',
+                source: ['/artist/lisa/'],
+                target: '/lisa/info',
+            },
+            {
+                title: 'LiSA Albums',
+                docs: 'https://docs.rsshub.app/other.html#lisa',
+                source: ['/artist/lisa/'],
+                target: '/lisa/disco',
+            },
+        ],
+    },
+    'lxixsxa.com': {
+        _name: 'LiSA Official',
+        www: [
+            {
+                title: 'LiSA News',
```

Comment:
```suggestion
                title: 'News',
```
title should match H3 heading of the docs